centos.org/ansible/templates/ssh_config.j2

```diff
@@ -1,10 +1,11 @@
-{% for host, value in cico_data.results.hosts.items() %}
+{% for host, value in cico_data.ansible_module_results.hosts.items() %}
 Host duffy_box
   HostName {{ value.hostname  }}
   User root
   Port 22
   StrictHostKeyChecking no
   PasswordAuthentication no
-  GlobalKnownHostsFile=/dev/null
+  GlobalKnownHostsFile /dev/null
+  IdentityFile /duffy-ssh-key/ssh-privatekey
```

Comment:
This is set in `/etc/ssh/ssh_config` on in the container, but as we use `ssh -F`, the global file isn't read :(

Reply:
Thinking out loud: can we concat `/etc/ssh/ssh_config` and this template to avoid hardcoding the path?

Reply:
Not trivially, but I can try a few things.

Reply:
The global config is btw here: https://github.com/centosci/images/blob/master/cico-workspace/ssh_config

Reply:
I think I've managed to make it nicely ;)